app/helpers/nav_helper.rb

```diff
@@ -6,35 +6,68 @@ def nav_items
     sample_period = Work::Period.new(community: current_community)
     sample_job = Work::Job.new(period: sample_period)
     sample_shift = Work::Shift.new(job: sample_job)
-    items = [
-      {
+    # Build up the main menu list
+    items = []
+    # check if there are any main menu items defined in Settings.
+    c_settings_topmenu = current_community.settings.top_menu_customizations
+    settings_topmenu = if c_settings_topmenu.nil?
+                         []
+                       else
+                         # create an array of arrays from the setting string.
+                         c_settings_topmenu.scan(%r{\[([\w\- ]+)\]\(([\w/\.:%&\?]+)\)})
+                       end
+    # add each standard main menu item unless that item has been overridden
+    # in the Settings.
+    unless settings_topmenu.any? { |h| h[0] == :people.to_s }
```

Comment:
Can we please match on the translated version of `:people`? See `config/locales/en/nav.yml`. That way this will still work if we introduce other languages later. It's also more intuitive because you'll be entering exactly what you see in the nav bar as the link name.

Reply:
This matching work can and should be done in the class. So in here you'd end up with like `menu_customization.has?(:people)`.